assets/js/common/Table/Table.jsx

```diff
@@ -134,7 +136,15 @@ function Table({
     })
     .reduce((d, filterFunction) => d.filter(filterFunction), data);
 
-  const sortedData = sortBy ? [...filteredData].sort(sortBy) : filteredData;
+  const searchKeys = columns
+    .filter(({ searchable = false }) => searchable === true)
+    .map(({ key }) => key);
+
+  const searchedData = searchBy
+    ? [...filteredData].filter((it) => searchByKey(it, searchBy, ...searchKeys))
```

Comment:
Curious why we spread on `searchKeys` here?

Reply:
Just because I started from the case of only having a single key